pkg/cmd/get_configaudit.go

```diff
@@ -43,10 +46,16 @@ NAME is the name of a particular Kubernetes workload.
 				return
 			}
 
+			selector := labels.Set{
+				kube.LabelResourceKind:      string(workload.Kind),
+				kube.LabelResourceName:      workload.Name,
+				kube.LabelResourceNamespace: workload.Namespace,
+			}.String()
+
 			kubectlCmd := exec.Command("kubectl",
 				"get",
 				starboard.ConfigAuditReportCRName,
-				fmt.Sprintf("-l=starboard.resource.kind=%s,starboard.resource.name=%s", workload.Kind, workload.Name),
+				fmt.Sprintf("-l=%s", selector),
```

Comment:
So functionally it returns the same info by doing a `.String()` on a `labels.Set()` call? Is there a test I could read up on?